### PR TITLE
Issue 21

### PR DESCRIPTION
**Pull request by jburel**
This PR should fix the build problem reported in #21

### CODE REVIEW

**Comment by pwalczysko** [comment]
repo2docker check failed

**Comment by pwalczysko** [comment]
Built locally. Kernel switched to the newly built conda env.

idr0002.ipynb - as expected

idr0002_save.ipynb:
1. the error with the `home` variable is not fixed -> fix was not propagated from idr0002.ipynb notebook
2. matplotlib not removed completely, the corresponding cell fails

idr0002_zarr.ipynb:
1. fails on ``ArrayNotFoundError: array not found at path `` on the cell ``%time data = load_dask_array_from_s3()``

**Comment by jburel** [comment]
@pwalczysko cf. zarr, I had some testing code for the cell painting study. This should not have been there.

**Comment by pwalczysko** [comment]
All three idr0002 notebooks now work as expected locally using conda.
I am still trying MyBinder, but not much success.

File: README.md [hunk 1]
```diff
@@ -11,23 +11,56 @@ This guide demonstrates how to use the CellProfiler Python API to analyze data s
...
+
+    $ conda activate omero-guide-cellprofiler
+
+The following steps are only required if you want to run the notebooks
+
+* If you have Anaconda installed:
+  * Start Jupyter from the Anaconda-navigator
+  * In the conda environment, run ``conda install ipykernel``
+  * To register the environment, run ``python -m ipykernel install --user --name omero-guide-cellprofiler``
+  * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:omero-guide-cellprofiler]``
```
Comment (pwalczysko) [inline README.md]: I do not have a menu item 
``Python [conda env:omero-guide-cellprofiler]``

Instead, I have just ``omero-guide-cellprofiler`` as a menu item (I call it ``test-jm-cp``) - this is a corresponding conda env naming :)


Screenshot 2023-05-02 at 12 34 59

Comment (pwalczysko) [reply README.md]: This is the same when using simply ``jupyter notebook`` from terminal or when using Anaconda Navigator - same menu items

**Comment by pwalczysko** [comment]
Binder crashed.

File: notebooks/idr0002_save.ipynb [hunk 2]
```diff
@@ -15,6 +15,18 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# When running the notebook on M1,\n",
+    "# you might have to remove this cell after removing the comment\n",
```
Comment (pwalczysko) [inline notebooks/idr0002_save.ipynb]: Not sure I understand the instruction. I should

1. remove the comment
2. run the cell
3. remove the cell 

?

**Comment by pwalczysko** [comment]
Line 45 of [idr0002_zarr](https://github.com/ome/omero-guide-cellprofiler/pull/22/files/5d203efc27753f6e4c04856271c0ca7665aa5d28..6e0046a43c3b8a9637f89ccf53676b11648a481e#diff-f0a73915d0751452122223f948978efeb7aa5d4eef3a8cf58d8e297742f7615c) has a cell which specifies a concrete path to the conda - looks like an omission


Screenshot 2023-05-02 at 15 52 04

**Comment by pwalczysko** [comment]
Other than https://github.com/ome/omero-guide-cellprofiler/pull/22#issuecomment-1531623294 lgtm, ready to merge after fixing of https://github.com/ome/omero-guide-cellprofiler/pull/22#issuecomment-1531623294

**Comment by jburel** [comment]
> Line 45 of [idr0002_zarr](https://github.com/ome/omero-guide-cellprofiler/pull/22/files/5d203efc27753f6e4c04856271c0ca7665aa5d28..6e0046a43c3b8a9637f89ccf53676b11648a481e#diff-f0a73915d0751452122223f948978efeb7aa5d4eef3a8cf58d8e297742f7615c) has a cell which specifies a concrete path to the conda - looks like an omission
> 
> Screenshot 2023-05-02 at 15 52 04

Are you sure you have the latest?
I have also improved the instructions

File: notebooks/idr0002_zarr.ipynb [hunk 3]
```diff
@@ -25,7 +37,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"JAVA_HOME\"]=\"/Users/jmarie/opt/anaconda3/envs/cp_test\""
```
Comment (pwalczysko) [inline notebooks/idr0002_zarr.ipynb]: Yes, I have the latest version, and this line persists

**Comment by pwalczysko** [comment]
The new instructions lgtm. But https://github.com/ome/omero-guide-cellprofiler/pull/22#discussion_r1182727989 is still there - it is also here in the github repo - not just in my local checkout.

File: notebooks/idr0002_zarr.ipynb (hunk 3 above)
Comment (jburel) [reply notebooks/idr0002_zarr.ipynb]: found it. sorted now

**Comment by jburel** [comment]
@pwalczysko sorted now

**Comment by pwalczysko** [comment]
Screenshot 2023-05-03 at 11 08 36
Yes, the cell is gone now, thanks a lot for the fixes !

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**